aptos-move/aptos-vm-types/src/output.rs

```diff
@@ -108,21 +108,34 @@ impl VMOutput {
         resolver: &impl AggregatorV1Resolver,
     ) -> anyhow::Result<TransactionOutput, VMStatus> {
         let materialized_output = self.try_materialize(resolver)?;
-        debug_assert!(
+        Self::convert_to_transaction_output(materialized_output)
+    }
+
+    /// Same as `try_materialize` but also constructs `TransactionOutput`.
```

Comment:
need to change this comment, this contructs `TransactionOutput` without materializing. (expecting nothing needs to be materialized)